internal/providers/msentraid/msentraid.go

```diff
@@ -175,9 +158,10 @@ func (p Provider) getGroups(token *oauth2.Token) ([]info.Group, error) {
 		if !ok || name == nil {
 			id := msGroup.GetId()
 			if id == nil {
```

Comment:
Btw, this isolated call for `GetId` is something that we did in a hurry to help a user debug some issues and we forgot to change it to the same format that we use in lines 176-180.